.travis.yml

```diff
@@ -8,6 +8,7 @@ python:
 
 os:
   - linux
+  - windows
```

Comment:
I think you'll need to change the language to `script` too, and install miniconda. You might just copy they way they do it: https://github.com/kangwonlee/travis-yml-conda-posix-nt/blob/master/.travis.yml

Reply:
In the end, we're not gonna do that, It makes no sense to test Windows if we have no dev to debug potential os specific failures. I'll just try autotune this once on my remote windows machine and we'll leave it at that.